### PR TITLE
Fix all the typos :D

### PR DESCRIPTION
**Pull request by jasikpark**
## Changes

- Fixes some typos in code and comments, I'm not sure this will pass immediately 😅

### CODE REVIEW

File: packages/compiler/browser/utils.ts [hunk 1]
```diff
@@ -102,11 +102,11 @@ function serializeAttributes(node: TagLikeNode): string {
   return output;
 }
 
-export interface SerializeOtions {
```
Comment (jasikpark) [inline packages/compiler/browser/utils.ts:105]: I believe this should be the only externally facing change?

**Comment by Princesseuh** [comment]
Was this done using automated tools? There was a PR recently to Astro core that fixed some typos using a tool, maybe it'd be interesting here too?

**Comment by jasikpark** [comment]
uhuh! https://github.com/marketplace/actions/typos-action

great tool

**Review by natemoo-re — APPROVED** [review]
Thanks for this!

**Pull request merged**